packages/Sortable/src/components/SortableItem/SortableItem.js

```diff
@@ -26,30 +26,35 @@ const SortableItem = ({ children, index, hasNumbers, onRemove }) => {
 
   return (
     <Draggable draggableId={`draggable-${children.props.sortId}`} index={index}>
-      {(provided, snapshot) => {
-        return (
-          <li
-            {...provided.draggableProps}
-            {...provided.dragHandleProps}
-            css={itemStyles}
-            data-is-dragging={snapshot.isDragging ? true : undefined}
-            isDragging={snapshot.isDragging}
-            ref={provided.innerRef}
-            aria-roledescription={I18n.t("sortable.aria_description")}
-          >
-            <div css={itemHandleStyles}>
-              <HandleIcon />
+      {(provided, snapshot) => (
+        <li
+          {...provided.draggableProps}
+          {...provided.dragHandleProps}
+          css={itemStyles}
+          data-component-type="sortable.item"
+          data-is-dragging={snapshot.isDragging ? true : undefined}
+          isDragging={snapshot.isDragging}
+          ref={provided.innerRef}
+          aria-roledescription={I18n.t("sortable.aria_description")}
+        >
+          <div css={itemHandleStyles} data-component-type="sortable.item.handle">
+            <HandleIcon />
+          </div>
+          {hasNumbers && (
+            <div css={itemIndexStyles} data-component-type="sortable.item.number">
+              {index + 1}
             </div>
-            {hasNumbers && <div css={itemIndexStyles}>{index + 1}</div>}
-            <div css={itemBodyStyles}>{children}</div>
-            {onRemove && (
-              <div css={itemCloseStyles}>
-                <Button.Close onClick={handleRemove} size="small" />
-              </div>
-            )}
-          </li>
-        );
-      }}
+          )}
+          <div css={itemBodyStyles} data-component-type="sortable.item.body">
+            {children}
+          </div>
+          {onRemove && (
+            <div css={itemCloseStyles} data-component-type="sortable.item.remove">
```

Comment:
Too much to add `data-component-type` throughout the DOM to allow precise control over styling? Or maybe just a different attribute name would be more appropriate since these aren't really _components_?

Reply:
maybe just `data-type`?

Reply:
Hm... `data-type` makes me think of arrays and hashmaps... maybe `data-element`? Or even `data-element-type` to maintain a similar naming pattern?

Reply:
ohhh I suggested data-type because I thought *data-component-type* was too long, but on that note, I would suggested `data-react-display-name=""` maybe? or just `data-display-name`?

Reply:
Oh. Yeah, both `component-type` and `display-name` are used in the issue https://github.com/acl-services/paprika/issues/101 so it's not clear what is actually being proposed.  

I like `component-type` better, but since we already set the `.displayName` for our components as a convention (https://github.com/acl-services/paprika/blob/master/packages/Sortable/src/Sortable.js#L87) maybe it's best to keep with the same name to reduce cognitive load? 

I was worried this might derail this PR, and it's not really that important at this point, but it's also a decision I would rather force now than just kick down the road.

Reply:
yes, let's keep it with `data-display-name` then.